common/create-amazon-catalog-item.adoc

```diff
@@ -0,0 +1,16 @@
+
+Use the following procedure to create an Amazon catalog item. Once created, the catalog item joins the service dialog with all of the options in the provisioning dialog; then, users can order Red Hat Enterprise Linux instances from the self-service catalog in the Service UI.
```

Comment:
I would recommend taking the second sentence here and breaking it into two sentences since it exceeds the recommended 25-word limit for a sentenced in the IBM guide. My only other recommendation here would be to replace the verb _join_ with a synonym like _combine_ or _link_. I had to read it a few times before I realized what it was conveying, but that could be me...I tend only to recognize the verb _join_ in a software context when referring to tables in a database.